mm-go-irckit/userbridge.go

```diff
@@ -392,6 +392,19 @@ func (u *User) handleReactionEvent(event interface{}) {
 		reaction = e.Reaction
 	}
 
+	if u.v.GetBool(u.br.Protocol() + ".hidereactions") {
+		logger.Debug("Not showing reaction: " + text + reaction)
+		u.saveLastViewedAt(channelID)
```

Comment:
maybe add a `defer u.saveLastViewedAt(channelID)` on line 394 and remove all the other `u.saveLastViewedAt`

Reply:
Fixed, thanks.